src/xenia/kernel/util/shim_utils.h

```diff
@@ -252,7 +252,7 @@ class PrimitivePointerParam : public ParamBase<uint32_t> {
   }
   T value() const { return *host_ptr_; }
   operator T() const = delete;
-  operator xe::be<T>*() const { return host_ptr_; }
+  operator xe::be<T> *() const { return host_ptr_; }
```

Comment:
Any reason for this change?

Reply:
I think that happened when I did xb format. I didn't change that myself. I'll push again without these changes.